src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -37,6 +41,22 @@
     }
 @end
 
+@private longRunningOptionalArrayMethodSampleCode(apiMethod)
```

Comment:
Since all PHP methods are optional array methods, I think you could remove the "OptionalArray" from the snippet function names.